src/modules/integrator-guide/pages/compose-configuration/duplication-detection.adoc

```diff
@@ -3,45 +3,98 @@ include::ROOT:partial$variables.adoc[]
-This is achieved by using rules to apply a criterion that determines how closely a field on a new or edited record matches the same field on an existing record.
+This is achieved by using rules to apply a criterion that determines how closely a field on a new or modified record matches the same field on an existing record.
 
-The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module, thereby preventing duplicates.
+The user interface displays a warning or an error message to alert the user of an existing record with  identical field values within a module, hence preventing the creation of duplicates.
 
-.Benefits of corteza duplicate detection:
-* Maintain data quality,
-* improves accuracy and efficiency in data,
-* enhances effective description of records,
-* optimizes data storage.
+.Benefits of using {PRODUCT_NAME} duplicate detection:
+* maintaining data quality,
+* Enhancing data accuracy and efficiency.
+* Enhancing effective description of records.
+* optimizes data storage
 
-== Validation Rules
+== Duplicate detection Rules
 
-The duplicate detection feature is enabled by using the value validation rules on a module.
-Validation rules verify that the designated fields in a record do not contain the same values, therefore ensuring that no duplicate values are saved when creating or editing a record.
+The duplicate detection feature is enabled by using validation duplicate detection rules on a module.
+The rules ensure that no duplicate values are recorded while creating or modifying a record by ensuring that the designated fields do not have the same values.
 
-Validation can be configured using two rules:
-
-* Strict duplicate value validation
+Duplicate detection validation can be configured using two rules:
 
+[cols="1s,5a"]
+|===
+| [#Strict-duplicate-validation]#<<Strict-duplicate-validation,Strict duplicate validation rule>>#
+|
 This rule ensures that a record will not be saved when a duplicate value is detected in the selected fields in any existing record of this module. 
 
-When the Strict duplicate value validation rule is applied, an error message pop-up will be displayed on the bottom right side of the screen as an indication that a record with the same values exists.
+When the Strict duplicate value validation rule is applied, a pop-up error message indicating the existence of a record with identical values will appear in the bottom right corner of the screen..
 
-* Soft duplicate value validation
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "duplication-detection/error.png",
+    "alias": "error",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 152,
+    "y": 42,
+    "w": 1568,
+    "h": 1080
+  },
+  "focus": {
+    "padding": "xs",
+    "x": 834,
+    "y": 450,
+    "w": 1204,
+    "h": 586
+  },
```

Comment:
The blur from the focus looks off; see here

![image](https://user-images.githubusercontent.com/12545711/198594064-f87f45d4-7d7d-4c42-b6bb-97963de2cb62.png)

In these cases, what I do, is I remove the `focus`, use that red box (see my comment in regards to image annotations) and use it to determine where the focus area should be.
Do the same for the other one as well.